components/settings/billing/subscription-actions.tsx

```
@@ -0,0 +1,340 @@
+	if (availableActions.length === 0) {
+		return null;
+	}
+
+	return (
+		<div className={cn('space-y-3', className)}>
+			{/* Actions Toggle */}
+			<div className="flex justify-end">
+				<button
+					onClick={() => setIsExpanded(!isExpanded)}
+					className="inline-flex items-center gap-2 px-3 py-1.5 text-sm font-medium text-slate-600 dark:text-slate-300 bg-slate-100 dark:bg-slate-700/50 rounded-lg hover:bg-slate-200 dark:hover:bg-slate-600/50 transition-colors"
+				>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)